### PR TITLE
Update QR generation sequence diagram

### LINKED ISSUE #213: Bug:

### PR DESCRIPTION
**Pull request by dillontkh**
New diagram:
QrAddPersonSequenceDiagram

Resolves #213

### CODE REVIEW

**Review by jjchee77 — APPROVED** [review]
LGTM!

File: docs/diagrams/QrAddPersonSequenceDiagram.puml [hunk 1]
```diff
@@ -17,7 +17,7 @@ activate Model
 Model -> Person : p.generateQrCode()
```
Comment (RyanNgWH) [inline docs/diagrams/QrAddPersonSequenceDiagram.puml:17]: Should the `p` here be removed as well?

**Review by RyanNgWH — CHANGES_REQUESTED** [review]

File: docs/diagrams/QrAddPersonSequenceDiagram.puml (hunk 1 above)
Comment (dillontkh) [reply docs/diagrams/QrAddPersonSequenceDiagram.puml:17]: Yes it should, good catch!

**Review by RyanNgWH — APPROVED** [review]
LGTM!

**Pull request merged**